src/main/scala/com/ing/wbaa/rokku/proxy/handler/namespace/NamespacesHandler.scala

```diff
@@ -39,15 +40,18 @@ trait NamespacesHandler {
     bucketCredentials.get(bucketName) match {
       case Some(credentials) =>
         logger.debug("credentials exist for bucket {}", bucketName.name)
+        incrementBucketNamespacesInCache()
```

Comment:
@kr7ysztof Initially reading this I thought that it will hold the number of namespaces in cache. The name is a bit misleading. This variable will hold the number of successful lookups in cache (cache hit). It is pretty common to use the hit/miss terminology for cache lookups. Maybe we can use something along the lines `incrementBucketNamespaceCacheHit/incrementBucketNamespaceCacheMiss`. This is just a suggestion I don't have a strong opinion on that I just noticed that initially I was a bit confused and I had to focus a bit more on that code part.

Reply:
yeah - your name is more meaningful